test/e2e/translation-banner.spec.js

```diff
@@ -21,9 +21,7 @@ test('Can close the translation banner', async ({ page }) => {
   await page.goto('/ru/search/')
   await page.click('[aria-label="Закрыть"]')
 
-  const banner = await page.locator(
-    '.span:has-text("Help us get to 100 percent")'
-  )
+  const banner = page.locator('.span:has-text("Help us get to 100 percent")')
```

Comment:
`page.locator` doesn't return a `Promise`. Completely unrelated to this PR but it was bugging me while I was debugging this particular test :sweat_smile: